dev/com.ibm.ws.microprofile.metrics.2.0/resources/com/ibm/ws/microprofile/metrics/resources/Metrics.nlsprops

```diff
@@ -98,6 +98,9 @@ cpu.systemLoadAverage.description=Displays the system load average for the last
 #Cpu processCpuLoad message
 cpu.processCpuLoad.description=Displays the 'recent cpu usage' for the Java Virtual Machine process.
 
+#Cpu processCpuTime message
+cpu.processCpuTime.description=Displays the 'recent cpu time' for the Java Virtual Machine process.
```

Comment:
is `recent cpu time` a distinct variable or element that is identified elsewhere?

Reply:
Ah. I've reworded the message to match [the documentation](https://docs.oracle.com/javase/7/docs/jre/api/management/extension/com/sun/management/OperatingSystemMXBean.html), similarly to how the processCpuLoad message does it so it makes more sense. Thanks.